docs/projects/version-control/index.md

```diff
@@ -5,68 +5,71 @@ position: 110
 
-![Branch-switcher UI](branch-switcher-ui.png "width=500")
+## Why use Config as Code?
 
-Of course, there is now a text representation of the process in the Git repository, and if you prefer editing text, open your favorite editor and go for it. We refer to the text format as Octopus Configuration Language (OCL), and it is very much inspired by [HCL](https://github.com/hashicorp/hcl).
+### Branch your code and deployments
 
-That means that where previously there was only a single current version of the deployment process, it is now possible to have many. When creating releases, the relevant branch can be selected. We have also added [branch system variables](docs/projects/variables/system-variables.md#release-branch-information) that can be used in your custom deployment scripts.
+Branches are Git's superpower, and we expose that power. Using Config as Code you can:
 
-:::warning
-Config-as-code only supports [git](https://git-scm.com/) repositories.  Before using this feature, you should be familiar with [git concepts](https://git-scm.com/doc) such as distributed version control, pushing, pulling, branching, merging, and fetching.
-:::
+* Switch branches, or create new ones, in the Octopus UI or your favorite Git client
+* Commit changes to your deployment process on a branch
+* Add (optional) commit messages when you change your deployment process, so others understand why it changed
+* Create releases from your branch, and deploy to test your changes, to support many versions of your deployment process
+* Evolve your deployments safely without worrying about breaking other teams' deployments
+* Roll back to a previous version of your deployment process if things go wrong
 
-### What's next?
+### Review, approve, and merge
 
-We have some strong opinions on what's next. The latest release of config-as-code supports _deployment processes_, _deployment settings_, and _non-sensitive variables_.  
+Review, approve, and merge your branches before you deploy to production. Getting more eyes across changes is always a good thing.
+Pull requests, protected branches, and code-owners enable a new set of workflows to improve the quality and safety of your releases. Reduce downtime from bad deployments and improve the quality of your releases.
+Audit and traceability
 
-We have plans for future releases of config-as-code to support Runbooks. In addition, we'd like to evolve the OCL schema to make it friendlier for editing by hand.
+The commit history improves the traceability of changes to a deployment process. Know what changed, when, by who, and most importantly why.
+No more searching through audit logs to find out why your configuration changed. Git history and diffs provide clear traceability and tell the complete story. Octopus even records the committer details in the Git history.
+Shared Git credentials
```

Comment:
Not sure if `Shared Git credentials` needs an additional space between this sentence so adding one in-case :smile: 

```suggestion
No more searching through audit logs to find out why your configuration changed. Git history and diffs provide clear traceability and tell the complete story. Octopus even records the committer details in the Git history.

Shared Git credentials
```